cmd/find-main.go

```diff
@@ -97,47 +97,52 @@ USAGE:
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
+UNITS
+   --smaller, --larger flags accept human-readable case-insensitive number suffixes
+   such as "k", "m", "g" and "t" referring to the metric units KB, MB, GB and TB
+   respectively. Adding an "i" to these prefixes, uses the IEC units, so that "gi"
+   refers to "gibibyte" or "GiB". A "b" at the end is also accepted. Without suffixes
+   the unit is bytes.
+
+   --older, --newer flags accept the suffixes "d", "w", "m" and "y" to refer to
+   units of days, weeks, months and years respectively. With the standard rate
+   of conversion being 7 days being in 1 week, 30 days in 1 month, and 365 days
```

Comment:
s/being 7 days being in 1 week/being 7 days in 1 week/

Reply:
Done

Reply:
The --older and --newer flags are not recognized. Throws a parse error
➜  mc git:(e9f0357b) ✗ mc find play/uploads   
play/uploads/moon-horizon-720p.jpg
➜  mc git:(e9f0357b) ✗ mc find play/uploads --smaller 1GB
play/uploads/moon-horizon-720p.jpg
➜  mc git:(e9f0357b) ✗ mc find play/uploads --larger 1GB 
➜  mc git:(e9f0357b) ✗ mc find play/uploads --older 1w  
mc: <ERROR> Error parsing string passed to flag older strconv.Atoi: parsing "": invalid syntax

Reply:
Oh we might need to fix that, i wonder what broke we tested this before @CarterMcClellan .

Reply:
We will have to fix this outside of this PR @poornas